### ISSUE TITLE
problem with Tooltip component and console.log

### ISSUE THREAD

**Issue by mits87** [KoRiGaN/Vue2Leaflet#138]
Hello, 

### Description
I'm using Vue2Leaflet for a short time and I have small problem.
When I added `Vue2Leaflet.Tooltip` component to my map and I'm using `@l-moveend` on map then I get many `console.logs`.

#### Live Demo
``` SCSS
  @import "~leaflet/dist/leaflet.css";

  #map {
    height: 800px;
  }
```

``` HTML
  <v-map id="map" ref="map" :zoom="zoom" :center="center" :options="options" @l-moveend="changeMapPosition">
    <v-tilelayer url="http://{s}.google.com/vt/lyrs=m&x={x}&y={y}&z={z}" :options="tile"></v-tilelayer>
    <v-marker v-for="marker in markers" :key="marker.id" :lat-lng="marker.point">
      <v-tooltip :content="marker.name" :options="{ permanent: true }"></v-tooltip>
    </v-marker>
  </v-map>
```

``` JS
  import Vue2Leaflet from 'vue2-leaflet';

  delete L.Icon.Default.prototype._getIconUrl;

  L.Icon.Default.imagePath = '';
  L.Icon.Default.mergeOptions({
    iconRetinaUrl: require('leaflet/dist/images/marker-icon-2x.png'),
    iconUrl: require('leaflet/dist/images/marker-icon.png'),
    shadowUrl: require('leaflet/dist/images/marker-shadow.png'),
  });

  export default {
    components: {
      'v-map': Vue2Leaflet.Map,
      'v-marker': Vue2Leaflet.Marker,
      'v-tooltip': Vue2Leaflet.Tooltip,
      'v-tilelayer': Vue2Leaflet.TileLayer,
    },

    data() {
      return {
        zoom: 16,
        center: [47.458356162688, 8.5555611641141],
        markers: [],
        tile: {
          maxZoom: 22,
          subdomains: ['mt0', 'mt1', 'mt2', 'mt3'],
        },
        options: {
          attributionControl: false
        },
      }
    },

    mounted() {
      this.changeMapPosition();
    },

    methods: {
      changeMapPosition() {
        const p = new Promise((resolve, reject) => {
          setTimeout(() => {
            resolve([
              {
                name: 'Hello',
                point: [47.458356162688, 8.5555611641141]
              },
              {
                name: 'World',
                point: [47.458216499999999, 8.548437383545]
              }
            ]);
          }, 1050);
        });

        p.then((markers) => {
          this.markers = markers;
        })
      }
    },
  }
```

#### Steps to Reproduce
If we are using: `<v-tooltip></v-tooltip>` component inside marker and listen on `l-moveend` event on map component.

#### Actual Results
[https://www.dropbox.com/s/3cbwhelxkgviynf/Screenshot%202018-03-14%2022.39.31.png](https://www.dropbox.com/s/3cbwhelxkgviynf/Screenshot%202018-03-14%2022.39.31.png)

### Browsers Affected
- [ x] Chrome
- [ x] Firefox
- [ x] Edge
- [ x] Safari 9
- [ x] Safari 8
- [ x] IE 11

### Versions
- Vue: v2.5.7
- Vue2Leaflet: v0.0.58

Any idea what's going one?

**Comment by KoRiGaN** [comment]
Hi @mits87,

This should be fixed in v0.0.60.

Hope this helps.

Micka

**Comment by KoRiGaN** [comment]
Closing as no comments for 3 weeks.
Please reopen this issue if it not solved yet.
